docker/Dockerfile.frontend

```diff
@@ -1,6 +1,5 @@
-FROM node:12.2.0-alpine
+FROM node:13.7.0-alpine
 WORKDIR /app
 ENV PATH /app/node_modules/.bin:$PATH
-COPY frontend/package.json /app/package.json
-COPY frontend/package-lock.json /app/package-lock.json
-RUN npm install
+COPY frontend /app
```

Comment:
So we are copying whole `node_modules` to the docker image? I don't think this is a good idea, maybe make `node_modules` as a volume so you wouldn't have to copy anything? I'm not sure

Reply:
Nope, do not copy node modules or do not set them as a volume. When building an image install packages from scratch with production flags and without dev dependencies.

Reply:
This is still the dev build, the production one will be pushed today. @pawelrubin asked to make it faster since rebuilding after modifying modules takes too much time. Turns out I tested the wrong compose yesterday and this one doesn't work properly as you mentioned. Going to delete this request.